test_util/test_aws_vpc.py

```diff
@@ -191,20 +154,39 @@ def main():
     vpc = None  # Set if the test owns the VPC
 
     if options.host_list is None:
-        log.info('CCM_VPC_HOSTS not provided, requesting new VPC from CCM...')
-        vpc = make_vpc(use_bare_os=options.test_install_prereqs)
-        host_list = vpc.hosts()
-    else:
```

Comment:
Ah, that was actually very intentional as i intended on making everything to be options.host_list because above it is parsed as a list:
options.host_list = os.environ['VPC_HOSTS'].split(',')

But, I flubbed making everything else options.host_list, so good catch!